src/FSharp.Control.TaskSeq.Test/TaskSeq.SkipWhile.Tests.fs

```diff
@@ -0,0 +1,332 @@
+module TaskSeq.Tests.skipWhile
+
+open System
+
+open Xunit
+open FsUnit.Xunit
+
+open FSharp.Control
+
+//
+// TaskSeq.skipWhile
+// TaskSeq.skipWhileAsync
+// TaskSeq.skipWhileInclusive
+// TaskSeq.skipWhileInclusiveAsync
+//
+
+exception SideEffectPastEnd of string
+
+[<AutoOpen>]
+module With =
+    /// The only real difference in semantics between the base and the *Inclusive variant lies in whether the final item is skipped.
+    let getFunction inclusive isAsync =
+        match inclusive, isAsync with
+        | false, false -> TaskSeq.skipWhile
+        | false, true -> fun pred -> TaskSeq.skipWhileAsync (pred >> Task.fromResult)
```

Comment:
pred made me thing predecessor, probably spell it in full?

Reply:
yeah. Or maybe I drop the whole thing. The `TaskSeq.takeXXX` has the same thing, but it rather confusing in practice.

Reply:
Done, dropped